code/obj/item/implant.dm

```diff
@@ -316,24 +316,30 @@ THROWING DARTS
 		if(inafterlife(src.owner))
 			return
 		DEBUG_MESSAGE("[src] calling to report crit")
+		SPAWN(rand(15, 25) SECONDS)
 		health_alert()
```

Comment:
Indent anything you want to occur inside the SPAWN, in the same way you indent anything you'd have inside an if()

Reply:
Alright, indented it.